openstack/resource_openstack_fw_group_v2.go

```diff
@@ -0,0 +1,363 @@
+package openstack
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/fwaas_v2/groups"
+)
+
+func resourceFWGroupV2() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceFWGroupV2Create,
+		ReadContext:   resourceFWGroupV2Read,
+		UpdateContext: resourceFWGroupV2Update,
+		DeleteContext: resourceFWGroupV2Delete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"tenant_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"ingress_firewall_policy_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"egress_firewall_policy_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"admin_state_up": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+
+			"status": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
```

Comment:
This is a computed attribute. Also please move it to the bottom.
```suggestion
			"status": {
				Type:     schema.TypeString,
				Computed: true,
			},
```